python/fledge/services/common/service_announcer.py

```diff
@@ -48,14 +48,14 @@ def __init__(self, sname, stype, port, txt):
         info = ServiceInfo(
             stype,
             service_name,
-            socket.inet_aton(host),
             port,
+            addresses=[socket.inet_aton(host)],
             properties=desc,
             server="{}.local.".format(host_name)
         )
         zeroconf = Zeroconf()
         # Refresh zeroconf cache
-        browser = ServiceBrowser(zeroconf, stype, handlers=[self.on_service_state_change])
+        # browser = ServiceBrowser(zeroconf, stype, handlers=[self.on_service_state_change])
```

Comment:
Zeroconf cache refresh not needed?

Reply:
It doesn't seem to be. In the testing I did it made no difference, but since I believe it was you that added it I meant to ask you if there was a particular case that I may have missed.

Reply:
It is used to browse for a service of a specific type.
The listener object will have its add_service() and remove_service() methods called when this browser discovers changes in the services availability.
Callback invoked by Zeroconf when new information arrives.Updates information required by browser in the Zeroconf cache.

However, this was even NOT correct before as `info` is NOT being updated when `ServiceStateChange.Added` case. See internal `on_service_state_change` def.